.github/workflows/ci-conan.yml

```diff
@@ -50,6 +51,7 @@ jobs:
             --options="${{ matrix.option_shared }}" \
             --update \
             --build=missing \
+            --build=b2/* \
```

Comment:
b2 from conancenter uses "modern" GLIBC and fails linking on our build. Rebuilding it until b2 supports `--build=compatible:[pattern]`.